client-mvp-04/src/sass/elements/_inputs.scss

```diff
@@ -42,3 +42,36 @@
     top: 14px;
   }
 }
+
+/****** CHECKBOX COMPONENT STYLING ******/
+
+input[type='checkbox'] {
+  display: none;
+}
+
+input[type='checkbox'] + span:hover {
+  border: 1px solid $dark-primary-color;
+}
+
```

Comment:
Do we have a hover effect on the checkbox? I didn't find it in the mockup.

Reply:
My mistake. I assumed the hover. I didn't realize it would've been detailed in FIGMA if desired. Removed it.